### ISSUE TITLE
v0.5.1 missing from npm

### ISSUE THREAD

**Issue by naugtur** [googleapis/nodejs-monitoring#130]
https://www.npmjs.com/package/@google-cloud/monitoring?activeTab=versions

**Comment by JustinBeckwith** [comment]
Greetings!  0.6.0 is available today - are you able to use that?  Or were you just observing that there was a 0.5.1 release in GitHub releases, and no relevant package?

**Comment by naugtur** [comment]
Yes, just pointing out that if you wished to release a fix, it's not available. Not an issue for me particularly.
Feel free to close if that was not a mistake.
